wsgicli.py

```diff
@@ -202,14 +204,20 @@ def import_from_path(import_path):
 def find_modules_from_path(import_path):
     import_from_path(import_path)
 
+    def parent_path(path):
+        return os.path.abspath(os.path.join(path, ".."))
+
+    site_dirs = site.getsitepackages()
```

Comment:
Wow... it's really great, I don't know such a useful function is in the standard libraries!